Services/TeamService.cs

```diff
@@ -0,0 +1,120 @@
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using Dapper;
+using Newtonsoft.Json;
+
+namespace CribblyBackend.Services
+{
+    public interface ITeamService
+    {
+        void Initialize();
+        Task<Team> GetById(int id);
+        void Update(Team Team);
+        Task Create(Team Team);
+        void Delete(Team Team);
+    }
+    public class TeamService : ITeamService
+    {
+        IDbConnection connection;
+        public TeamService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        public async Task Create(Team Team)
+        {
+            await connection.ExecuteAsync(
+                @"INSERT INTO Teams(
+                    Id, 
+                    Name, 
+                    Division, 
+                    Players, 
+                    GameScores, 
+                    PlayInGames, 
+                    BracketGames, 
+                    Wins, 
+                    Losses, 
+                    TotalScore, 
+                    Ranking, 
+                    Seed, 
+                    InTournament
+                )
+                VALUES (
+                    @Id, 
+                    @Name, 
+                    @Division, 
+                    @Players, 
+                    @GameScores, 
+                    @PlayInGames, 
+                    @BracketGames, 
+                    @Wins, 
+                    @Losses, 
+                    @TotalScore, 
+                    @Ranking, 
+                    @Seed, 
+                    @InTournament
+                )",
+                new { 
+                    Id = Team.Id, 
+                    Name = Team.Name, 
+                    Division = Team.Division, 
+                    Players = JsonConvert.SerializeObject(Team.Players), 
+                    GameScores = JsonConvert.SerializeObject(Team.GameScores), 
+                    PlayInGames = JsonConvert.SerializeObject(Team.PlayInGames), 
+                    BracketGames = JsonConvert.SerializeObject(Team.BracketGames), 
```

Comment:
We'll have to put more thought into our db design here. Since we already have a Players table and will presumably have a Games table, we don't have to store this info in the Teams table. Instead, we can implement a one-to-many mapping using Dapper and some joins in the query. [This blog post](https://www.tritac.com/nl/blog/dappernet-by-example/) is a good place to start.

Reply:
Fixed. Removed JSON and added Dapper object mapping